test/HeywoodHealthcareTest.js

```diff
@@ -0,0 +1,134 @@
+
+        const afterTime = moment();
+
+        const expectedAvailability = false;
+        expect(results.individualLocationData[0].hasAvailability).is.deep.equal(
+            expectedAvailability
+        );
+
+        expect(results.individualLocationData[0].availability).is.deep.equal(
+            {}
+        );
+        /*
+        Structure conformance expectations:
+
+        - All the timestamps are expected to be between before
+            and after when the scraper was executed
+        - Each site's results object must have a property named "hasAvailability"
+         */
+        expect(
+            moment(results.individualLocationData[0].timestamp).isBetween(
+                beforeTime,
+                afterTime
+            )
+        );
+
+        // console.log(`${JSON.stringify(results)}`);
+
+        if (process.env.DEVELOPMENT) {
+            file.write(
+                `${process.cwd()}/out_no_browser.json`,
+                `${JSON.stringify(results, null, "   ")}`
+            );
+        }
+    });
+
+    it("should show several slots", async function () {
+        const results = await scraper(false, availabilityFetchService);
+
+        const afterTime = moment();
+
+        const expectedAvailability = true;
+        expect(results.individualLocationData[0].hasAvailability).is.deep.equal(
+            expectedAvailability
+        );
+
+        const date = Object.keys(
+            results.individualLocationData[0].availability
+        )[0];
+
+        expect(
+            results.individualLocationData[0].availability[date]
+                .numberAvailableAppointments
+        ).equals(116);
+        /*
+        Structure conformance expectations:
+
+        - All the timestamps are expected to be between before
+            and after when the scraper was executed
+        - Each site's results object must have a property named "hasAvailability"
+         */
+        expect(
+            moment(results.individualLocationData[0].timestamp).isBetween(
+                beforeTime,
+                afterTime
+            )
+        );
+
+        // console.log(`${JSON.stringify(results)}`);
+
+        if (process.env.DEVELOPMENT) {
+            file.write(
+                `${process.cwd()}/out_no_browser.json`,
+                `${JSON.stringify(results, null, "   ")}`
+            );
+        }
+    });
+});
+
+/* utilities */
+
+/** Generator to feed filenames sequentially to the "with availability" test. */
+function* noAvailabilityFilenames() {
+    yield "firstPage-noSlots-hasMoreTimes-scripts-removed.html";
+    yield "secondPage-noSlots-scripts-removed.html";
+}
+const noAvailabilityFilenameGenerator = noAvailabilityFilenames();
+
+const noAvailabilityFetchService = {
+    async fetchAvailability(/*site*/) {
+        return loadTestHtmlFromFile(
+            "HeywoodHealthcare",
+            noAvailabilityFilenameGenerator.next().value
```

Comment:
`testFolderName` should be `"HeywoodHealthCare"`.  **(capitalize the C)**.
I'm running the tests on Linux which has case-sensitive filenames.

Reply:
Now I'm thinking that you should probably rename the directory instead.  The scraper is called "HeywoodHealthcare" (with only the H's capitalized), but the test folder is HeywoodHealthCare (with HHC capitalized).

Also, the real place calls itself "Heywood Healthcare"

Reply:
Yeah, I thought I had eradicated the "C" in all references to the site after I noticed that they didn't capitalize it. Apparently the test resources folder evaded my notice.